server.js

```diff
@@ -0,0 +1,16 @@
+// Used to start the site server
```

Comment:
It looks good. Mine with angular is almost the same: https://github.com/Volunteero/volunteero-front-end-base/blob/master/server.js.